rpc_jobs/hw_multi_node_aio.yml

```diff
@@ -139,21 +158,23 @@
                   "DEPLOY_MAAS=no"
                   ]
               ) // deploy_sh
-              node(){{
-                maas.prepare(instance_name: instance_name)
-                maas.deploy()
-                maas.verify()
-                tempest.tempest("deploy1")
-                kibana.kibana(env.KIBANA_SELENIUM_BRANCH, "deploy1")
-                holland.holland()
-              }}
-              ssh_slave.destroy()
-              maas.entity_cleanup(instance_name: instance_name)
+              maas.prepare(instance_name: instance_name)
+              maas.deploy()
+              maas.verify()
+              tempest.tempest()
+              kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+              holland.holland()
             }} // hw node
           }} catch (e){{
               print e
               throw e
           }} finally {{
+              ssh_slave.destroy()
```

Comment:
its not obvious that these two calls of ssh_slave.destroy() do different things as there are no different parameters. As noted in another comment, I think destroy should be refactored to take a name argument.